saw-remote-api/Dockerfile

```diff
@@ -47,26 +47,23 @@ RUN useradd -m saw
 COPY --chown=saw:saw . /home/saw
 USER saw
 WORKDIR /home/saw
-ARG CRYPTOLPATH="/cryptol/.cryptol"
 ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8
 COPY cabal.GHC-8.8.4.config cabal.project.freeze
-RUN mkdir -p rootfs/usr/local/bin
-RUN cabal v2-update && \
-    cabal v2-build -j saw-remote-api:exe:saw-remote-api && \
-    cp $(cabal v2-exec which saw-remote-api) rootfs/usr/local/bin
-RUN mkdir -p rootfs/"${CRYPTOLPATH}" \
-    && cp -r deps/cryptol/lib/* rootfs/"${CRYPTOLPATH}"
+RUN cabal v2-update && cabal v2-build saw-remote-api
```

Comment:
Dropping `-j` will lock this build to only one core, which could make it take a really long time to build.

Reply:
Cool - I'll add it back in. Thanks!